projects/plugins/boost/app/assets/src/js/react-components/BoostPricingTable.tsx

```diff
@@ -38,6 +38,41 @@ const supportContext = __(
 	'jetpack-boost'
 );
 
+const automaticallyUpdatedContext = sprintf(
+	'%1$s<br><br>%2$s',
+	__(
+		'It’s essential to regenerate Critical CSS to optimize your site speed whenever your HTML or CSS structure changes. Being on top of this can be tedious and time-consuming.',
+		'jetpack-boost'
+	),
+	__(
+		'Boost’s cloud service can automatically detect when your site needs the Critical CSS regenerated, and perform this function behind the scenes without requiring you to monitor it manually.',
+		'jetpack-boost'
+	)
+);
+
+const manuallyUpdatedContext = sprintf(
+	'%1$s<br><br><ul><li>%2$s</li><li>%3$s</li><li>%4$s</li><li>%5$s</li><li>%6$s</li><li>%7$s</li></ul>',
```

Comment:
😄 Done in https://github.com/Automattic/jetpack/pull/29250/commits/c850ab5cccea29a30acc15c2e9f1925b8b785853.

Reply:
That's better, but we can do even better-er. Because this is a `tsx` file, you can intersperse HTML like syntax, and it will build react-friendly elements at compile time. Try replacing this with:

```
const manuallyUpdatedContext = <span>
	{ __(
		'To enhance the speed of your site, with this plan you will need to optimize CSS by using the Manual Critical CSS generation feature whenever you:',
		'jetpack-boost'
	) }
	<br /><br />
	<ul>
		<li>{ __( 'Make theme changes.', 'jetpack-boost' ) }</li>
		<li>{ __( 'Write a new post/page.', 'jetpack-boost' ) }</li>
		<li>{ __( 'Edit a post/page.', 'jetpack-boost' ) }</li>
		<li>
			{
				__(
					'Activate, deactivate, or update plugins that impact your site layout or HTML structure.',
					'jetpack-boost'
				)
			}
		</li>
		<li>
			{
				__(
					'Change settings of plugins that impact your site layout or HTML structure.',
					'jetpack-boost'
				)
			}
		</li>
		<li>
			{
				__(
					'Upgrade your WordPress version if the new release includes core CSS changes.',
					'jetpack-boost'
				)
			}
		</li>
	</ul>
</span>;
```

Then you no longer need a linter comment, and you can just use it in the PricingTableItem like so:
```
tooltipInfo={ manuallyUpdatedContext }
```